README.md

```diff
@@ -1,3 +1,7 @@
 # ECE4437-Robot
 
-Caleb Ballard, Aruba Bukhari
\ No newline at end of file
+Caleb Ballard, Aruba Bukhari
+
+## Intro
+
+This robot will follow a path
```

Comment:
I don't like this description, make it more detailed

Reply:
:(